File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while ((n < 1) || (n > 8)) ;

    // for each row
    for (int i = 0; i < n; i++)
    {
        //spaces before the blocks to make middle gap even
        for (int j = 0; j < n - i - 1; j++)
        {
            printf(" ");
        }
        // for each column
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        // space between blocks
        printf("  ");
        // for each column
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        // move to next row
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

# TODO

while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

i = 1
while i <= n:

    # spaces before the blocks to make them even
    print(" " * (n - i), end="")

    # left blocks
    print("#" * i, end="")

    # middle gap
    print("  ", end="")

    # right blocks
    print("#" * i, end="")

    # move to next row
    print()

    i += 1